RefRed/preview_config/preview_config.py

```diff
@@ -119,8 +111,11 @@ def check_format(self):
         # system data
         _system_table = empty((len(self.system_name)), dtype=object)
         for _index, _name in enumerate(self.system_name):
-            _value = str(_dom.getElementsByTagName(_name)[0].childNodes[0].data)
-            _system_table[_index] = str(_value)
+            try:
+                _value = str(_dom.getElementsByTagName(_name)[0].childNodes[0].data)
+                _system_table[_index] = str(_value)
+            except:
+                _system_table[_index] = ""
```

Comment:
is it OK to be permissive here? If is OK if any of `"instrument_name", "timestamp", "version"` are missing ?

Reply:
I think that for a preview it's a good idea to be permissive, since the info will be displayed for inspection. That may help with sorting out problems.